test/integration/puppetlabs/services/certificate_authority/certificate_authority_int_test.clj

```diff
@@ -167,6 +170,41 @@
                 (is (= 403 (:status response))
                     (ks/pprint-to-string response))))))))))
 
+(deftest ^:integration certificate-with-ca-true-extension-refused
+  (testing (str "Validates that the server rejects a csr for signing"
+                " that has the v3 CA:TRUE extension")
+    (let [master-conf-dir (str test-resources-dir "/ca_true_test/master/conf")
+          req-dir (str master-conf-dir "/ssl/ca/requests")
+          key-pair (ssl-utils/generate-key-pair)
+          subjectDN (ssl-utils/cn "test_cert_ca_true")
+          serial 1
+          public-key (ssl-utils/get-public-key key-pair)
+          ca-ext (ca/create-ca-extensions subjectDN
+                                          serial
+                                          public-key)
+          csr (ssl-utils/generate-certificate-request key-pair
+                                                      subjectDN
+                                                      ca-ext)]
+          (fs/mkdir req-dir)
+          (ssl-utils/obj->pem! csr (str req-dir
+                                        "/test_cert_ca_true.pem"))
```

Comment:
nitpick: I think the indentation on these two lines is off - since they're not values in the let block, the should be at the same level of indentation as the `(bootstrap/with-puppetserver-running` below.